person.rb

```diff
@@ -30,6 +30,10 @@ def can_use_services?
   def correct_name
     @name
   end
+
```

Comment:
- [ ] great job. ensure that Getters and Setters for @rentals should be present
eg `attr_accessor :name, :age, :rentals`

- [ ] please `@rentals should be initialized.`.. you can learn more [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars)